sample/axpy.py

```diff
@@ -28,6 +28,7 @@ def cpu_axpy(a):
 import ctypes
 def gpu_axpy(res):
     source = """
+    #include <hip/hip_runtime.h> // Needed for older ROCm versions (at least 4.2) to access blockDim / blockIdx / threadIdx
```

Comment:
This include on newer versions causes errors.
It can be hidden inside a #ifdef
```cpp
#if (HIP_VERSION_MAJOR == 4 && HIP_VERSION_MINOR <= 2)
#include <hip/hip_runtime.h> // Needed for older ROCm versions (at least 4.2)
#endif
```

I do not have a 4.2 rocm installation.
Can you try it?

Reply:
Will try out

Reply:
This works for me

Reply:
fixed